fighthealthinsurance/settings.py

```diff
@@ -356,8 +374,30 @@ class TestSync(Dev):
 
 
 class Prod(Base):
+    sentry_endpoint = os.getenv("SENTRY_ENDPOINT")
+
+    if sentry_endpoint:
+
+        import sentry_sdk
+
+        sentry_sdk.init(
+            dsn=sentry_endpoint,
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for tracing.
+            traces_sample_rate=1.0,
+            _experiments={
+                # Set continuous_profiling_auto_start to True
+                # to automatically start the profiler on when
+                # possible.
+                "continuous_profiling_auto_start": True,
+            },
+        )
+
```

Comment:
Unless I'm misreading the diff, this block is inside a class definition. Class definitions execute immediately at runtime, so this isn't only going to run in production. It seems odd to me to have kind of logic directly in the class definition instead of within a function. Do you think that we should move this to another module where we can make it so that **SENTRY_ENDPOINT** is a regular Django setting and then reference that outside of the settings file for setup?

Reply:
oooh yeah we could do that, but we'd want to do it at app startup yeah.

Reply:
But also these changes are included by accident :p